spec/system/candidate_interface/candidate_eligibility_spec.rb

```diff
@@ -16,10 +18,17 @@ def when_i_click_start_on_the_start_page
     click_on t('application_form.begin_button')
   end
 
+  def and_i_press_continue
+    click_on 'Continue'
+  end
+
+  def then_i_see_validation_errors
+    expect(page).to have_content 'Choose if you are a citizen of the UK, EU or EEA'
```

Comment:
Nit, can merge without

```suggestion
    expect(page).to have_content t('activemodel.errors.models.candidate_interface/eligibility_form.attributes.eligible_citizen.blank')
```

(please double check I typed it in correctly)